lib/pwauth/okta/impl.go

```diff
@@ -5,35 +5,74 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"time"
 
 	"github.com/Cloud-Foundations/Dominator/lib/log"
 )
 
 const (
-	authPath           = "/api/v1/authn"
-	authEndpointFormat = "https://%s.okta.com" + authPath
+	authPath               = "/api/v1/authn"
+	authEndpointFormat     = "https://%s.okta.com" + authPath
+	factorsVerifyPathExtra = "/factors/%s/verify"
 )
 
-type loginDataType struct {
+type VerifyTOTPFactorDataType struct {
+	StateToken string `json:"stateToken,omitempty"`
+	PassCode   string `json:"passCode,omitempty"`
+}
```

Comment:
Please convert all JSON names from camelCase to lower_case. This comment applies for all types.

Reply:
This is the okta API... this cannot be changed.